src/main/resources/application.yml

```diff
@@ -2,6 +2,9 @@ kafka:
   brokers: "kafka:9092"
 
 importer:
+  masking:
```

Comment:
We need enable disable flag for this feature

Reply:
done